ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -128,15 +127,6 @@
       - "{{ docker_images_group1 }}"
       - "{{ docker_images_group2 }}"
 
-  - name: Remove {{ oc_host_config_dir }}/master/master-config.yaml when resetting cluster
```

Comment:
I think we want this block back in, on a reset it helps to be sure we are regenerating the master-config.yaml.
I'm running reset_environment.sh now, and it still has changes in master-config.yaml from prior run.